.github/workflows/format.yml

```diff
@@ -22,7 +22,7 @@ jobs:
           find . -name '*.m' -exec clang-format -i {} \;
           find . -path '*/ios/**/*.h' -exec clang-format -i {} \;
           find . -path '*/macos/**/*.h' -exec clang-format -i {} \;
-          git diff --exit-code
+          git diff --exit-code || (git commit --all -m "Clang Format" && git push)
```

Comment:
Nice one :)

This and the Swift one look useful enough on their that i'll try to get this on the `master` branch separately